### PR TITLE
test: refactor jest test negative test cases - phase 1

### LINKED ISSUE #3483: test: refactor jest test negative test cases - phase 1
Refactor all the remaining negative test case exception assertions for all tests in cacti/packages so that they do not use try-catch blocks but instead declare the requirements through Jest's (and jest-extended's) own API.

The parent issue with a much more detailed explanation of what's needed and code examples on how to do it:
Parent Issue: https://github.com/hyperledger/cacti/issues/3455

Acceptance Criteria
Test passes the same as before
Negative test case assertions have been migrated the way it is explained in the parent issue.

### PR DESCRIPTION
**Pull request by ashnashahgrover**
## **Commit** to be reviewed
test: refactor jest test negative test cases - phase 1
```
Primary Changes
---------------
1. Refactored all remaining negative test case exception assertions under cacti/packages Removed try-catch blocks, replaced with declarations through jest-extended's own API.
```
Fixes #3483

### CODE REVIEW

**Comment by ashnashahgrover** [comment]
@jagpreetsinghsasan 

Should I make a issue to investigate the inconsistencies in deploy-contract-from-json-xdai.test.ts and make this issue dependent on that?

**Review by petermetz — COMMENTED** [review]
> @jagpreetsinghsasan 
> Should I make a issue to investigate the inconsistencies in deploy-contract-from-json-xdai.test.ts and make this issue dependent on that? 

@ashnashahgrover Yes please! It's looking good in general but we need to make sure the test assertions are working as intended (and if we found an underlying bug then first we have to fix those in order to be able to prove that the asesrtions themselves work as intended)

File: packages/cactus-verifier-client/src/test/typescript/unit/verifier-factory.test.ts [hunk 1]
```diff
@@ -49,6 +51,11 @@ describe("Constructor Tests", () => {
 });
 
 describe("getVerifier Tests", () => {
+  const log = LoggerProvider.getOrCreate({
+    label: "geth-contract-deploy-and-invoke-using-keychain-v1.test.ts",
```
Comment (outSH) [inline packages/cactus-verifier-client/src/test/typescript/unit/verifier-factory.test.ts:55]: Please fix the log label

**Review by outSH — CHANGES_REQUESTED** [review]
LGTM, I only found one small things to fix

File: packages/cactus-verifier-client/src/test/typescript/unit/verifier-factory.test.ts (hunk 1 above)
Comment (ashnashahgrover) [reply packages/cactus-verifier-client/src/test/typescript/unit/verifier-factory.test.ts]: Fixed this.

**Review by outSH — APPROVED** [review]
LGTM, thank you!

**Review by petermetz — APPROVED** [review]

**Comment by petermetz** [comment]
Closing this due to inactivity, please feel free to re-open anytime if/when bandwidth is available to take it to the finish line (some of the checks are failing such as the linter)

**Pull request closed without merging**